lib/presentation/profile/profile_screen.dart

```diff
@@ -21,9 +24,10 @@ class _UserProfilePageState extends State<UserProfilePage> {
   @override
   Widget build(BuildContext context) {
     final bioController = TextEditingController();
+    final profileBloc = getIt<ProfileBloc>();
```

Comment:
If we don't pass it to another screen or widget, then we shouldn't initialize it in the build method and should just use it.

Reply:
This change actually seemed to fix the issue of the profile not being reloaded after login. (`profileBloc` is used after pushing the auth route)